osmaxx/excerptexport/admin.py

```diff
@@ -35,6 +53,36 @@ class OutputFileAdmin(admin.ModelAdmin):
 
 @admin.register(Export)
 class ExportAdmin(admin.ModelAdmin):
-    list_display = ('id', 'file_format', 'conversion_service_job_id', 'status', 'finished_at')
+    list_display = ('id', 'file_format', 'conversion_service_job_id', 'status', 'created_at', 'finished_at')
     list_display_links = ('id', )
-    list_filter = ['finished_at', 'created_at', 'updated_at']
+    list_filter = ['status', 'finished_at', 'created_at', 'updated_at']
+    fields = ('file_format', 'conversion_service_job_id', 'status', 'link_to_extraction_order', 'finished_at', 'created_at', 'updated_at')
+    readonly_fields = ('finished_at', 'created_at', 'updated_at', 'link_to_extraction_order')
+
+    def link_to_extraction_order(self, obj):
+        link = urlresolvers.reverse("admin:excerptexport_extractionorder_change", args=[obj.extraction_order.id])
+        return u'<a href="%s">%s</a>' % (link, obj.extraction_order.excerpt_name)
+    link_to_extraction_order.allow_tags = True
+
+
+User = get_user_model()
+admin.site.unregister(User)
+
+
+class ExtractionOrderInline(admin.TabularInline):
+    model = ExtractionOrder
+    fields = ('id', 'excerpt', 'orderer')
+    readonly_fields = fields
+    can_delete = False
+    show_change_link = True
+    extra = 0
+
+    def has_add_permission(self, request):
+        return False
+
+
+@admin.register(User)
+class UserAdmin2(UserAdmin):
+    inlines = [
```

Comment:
As `UserAdmin` might declare its own `inlines` in the future, should this be
```python
    inlines += [
        # ...
```
or would we _want_ to override pre-existing `inlines` in that case?